concordia/templates/django_registration/activation_complete.html

```diff
@@ -6,7 +6,7 @@
         <div class="col-md-8 mx-auto p-3">
             <h2>Account Activation Complete</h2>
             <p>
-                Welcome back! Your registration is now complete. Go ahead and <a href="{% url 'login' %}">Login</a>.
+                Welcome back! Your registration is now complete and you are logged in.
```

Comment:
@elainekamlley @lalgee Would like suggestions for better wording on this page. The user has just clicked their email activation link and is already logged in. Should we put a link to the welcome guide?

Reply:
Yes!  Linking to Welcome Guide is great also a let's go link.  Suggested language

Welcome back!  Your registration is now complete and you are logged in.  

Visit the By the People Welcome Guide link) for instructions and help getting started.  Or ,jump right in (link to 1st page in need of transcription) to a page that needs your help!